### ISSUE TITLE
Can'e include theme in posts

### ISSUE THREAD

**Issue by xiadingZ** [plusjade/jekyll-bootstrap#301]
I create a post using `rake post title="test"`,  this is my test code:

`---
layout: page
title: "test"
description: "this is description"
category: prml
tags: [book, prml]
---
{% include JB/setup %}

# test one

### test three

$$x=\frac{-b\pm\sqrt{b^2-4ac}}{2a}$$
`

This file is in _posts/, but when I open it in browser, it doesn't have theme as other webpages in root directory, it only has text. then I try to create a new post and copy the content of my ./about.md to it . However, the about.md in my root directory has theme, but that in _posts/ doesn't.  

How to make all my posts have same theme as webpage in my root directory.

**Comment by alexsanjoseph** [comment]
:+1:

**Comment by Q-Mart** [comment]
I was having the same problems and after reading [this](https://github.com/plusjade/jekyll-bootstrap/issues/290), I managed to resolve it.

Basically `_includes/JB/setup`, replace line 20 with `{% capture ASSET_PATH %}{{ BASE_PATH }}/assets/themes/{{ layout.theme.name }}{% endcapture %}` (so just change `page.theme.name` to `layout.theme.name`)

**Comment by Matthew1994** [comment]
Yes, I meet that problem too!But it finally works after correcting the codes Q-Mart mentioned!
